### ISSUE TITLE
Emoji picker icon is annoying

### ISSUE THREAD

**Issue by M1dgard** [tootsuite/mastodon#6063]
tl;dr: That "tears of joy" emoji really really really annoys me. Please change the picker icon to something—anything—else.

![screenshot_2017-12-18_13-42-00](https://user-images.githubusercontent.com/4283376/34106651-52528a1e-e3f9-11e7-8c2e-fd00a6285238.png)

Due to its usage on things that aren't remotely funny, I am conditioned to immediately look the other way when I see it. I don't want to see it next to every post I type. I don't want to write that my cat has died next to and and I don't want to click it to select a sad emoji.

Any other emoji would be better.

I have now blocked my instances' `1f602.svg` in my browser, which prevents me from using the emoji picker (though that's not really a big deal for me). There's nothing left to click on, related to #5743? (The area where it used to be doesn't change my cursor either, and doesn't respond to clicks.)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by m4sk1n** [comment]
😂

**Comment by m4sk1n** [comment]
I wish I would only have this kind of problems. Does it look better for you?
![Uploading Screenshot_20171218-153142.jpg…]()

**Comment by m4sk1n** [comment]
I wish I would only have this kind of problems. Does it look better for you?
![screenshot_20171218-153142](https://user-images.githubusercontent.com/21127288/34111278-8ef49d6c-e409-11e7-92a8-86645951ed97.jpg)

**Comment by M1dgard** [comment]
@m4sk1n Looking better, but I think that the very subtle reference to suicide might distress some people. Also I'm not convinced that the two icons work, it seems too wide. Maybe something like this:

![screenshot_2017-12-18_18-44-03](https://user-images.githubusercontent.com/4283376/34119760-a14983fa-e423-11e7-8272-72d50e2573cb.png)

**Comment by Cassolotl** [comment]
I think I saw a site somewhere, where the emoji that represented the emoji picker was a different randomly selected emoji each time the page reloaded. That could be nice - maybe it could be limited to just the "faces" section, for clarity?

**Comment by m4sk1n** [comment]
@M1dgard maybe, to support diversities, the middle finger should be picked randomly, from every skin tone… such a racist…

**Comment by ProgVal** [comment]
@m4sk1n Could you avoid this kind of jokes? (suicide / fake racism claim)

**Comment by chriswmartin** [comment]
@Cassolotl The way Discord handles it is really nice in my opinion, every time you mouse over the emoji picker icon it changes to a new random emoji.

**Comment by joyeusenoelle** [comment]
The aggressive suggestions for default emoji are *incredibly* alarming. Of *course* neither of those suggestions will be implemented in default Mastodon.

**Comment by averageunicorn** [comment]
I don't have an opinion on changing the emoji picker icon, but this userstyle might help those who dislike the current or any future one: https://userstyles.org/styles/141756/mastodon-change-emoji

**Comment by m4sk1n** [comment]
@ProgVal, it’s just irony, I don’t think that something like “😂” can annoy someone just because of his/her nad mood… The “racist claim” was used to show, that everyone can find problem in literally everything, if (s)he wants to find it.

**Comment by nightpool** [comment]
I've deleted a couple comments to keep this thread on topic. Please remember to keep Mastodon's code of conduct in mind when using the issue tracker and forums. 

https://github.com/tootsuite/mastodon/blob/master/CODE_OF_CONDUCT.md

**Comment by Gargron** [comment]
The 😂 emoji is very symbolic. It's also merely a part of the UI that you stop noticing once you're used to Mastodon. In your described scenario, you may also take issue with the button below saying "toot", or the placeholder text asking "What is on your mind?" instead of something comforting, or the elephant friend illustration smiling.

**Comment by M1dgard** [comment]
@Gargron To be clear, it's not because of Mastodon that I take issue with it, but because of videos like this. The bar of emojis sit at the top while a video plays underneath.

![screenshot_2017-12-22_14-58-30](https://user-images.githubusercontent.com/4283376/34300669-5f51168a-e729-11e7-8f45-1cb6a6265f6a.png)
